install-vphaser.sh

```diff
@@ -0,0 +1,72 @@
+#          
+#        VPhaser/VProfiler install script
+# 
+#	 This script will install VPhaser and VProfiler in your current directory.
+#  To run it, type:
+#
+#    bash install-vphaser.sh
+#
+#  The zipped package file from the Broad Institute is required to run this script.
+#  See the accompanying file vphaser-info.txt for more information.
+#
```

Comment:
May be good to printf this (that the zipped package is required to run the script)